junifer/datagrabber/utils.py

```diff
@@ -51,15 +51,7 @@
         "mandatory": ["pattern", "space"],
         "optional": [],
     },
-    "probseg_CSF": {
-        "mandatory": ["pattern", "space"],
-        "optional": [],
-    },
-    "probseg_GM": {
-        "mandatory": ["pattern", "space"],
-        "optional": [],
-    },
-    "probseg_WM": {
+    "VBM_CSF": {
```

Comment:
Are you sure about this that `VBM` is in fact `probseg`?

Reply:
Yes. It is the tissue probability map.